docs/basics/env-var-info.md

```diff
@@ -1,147 +1,161 @@
 # .env Variable Info
 
 ## COMPOSE_HTTP_TIMEOUT
-
 * Default value: `60`
+
 Description:
-See <https://docs.docker.com/compose/reference/envvars/#compose_http_timeout>
+See [Compose HTTP Timeout](https://docs.docker.com/compose/reference/envvars/#compose_http_timeout)
 This can be increased if you are seeing timeout issues when running compose. It is uncommon to need to adjust this option, but has been seen most commonly on very low powered CPU systems (older raspberry pi) or systems with failing storage (hard drives).
 
 ## DOCKERCONFDIR
-
 * System Detected value: `~/.config/appdata`
+
```

Comment:
Lists don't need an empty line between each item. Just an empty line before the list and after the list.